docs/src/commands/build-commands.md

```diff
@@ -1 +1,10 @@
 # Build Commands
+
+Build commands assist implementation projects with generating code and
+documentation from Buffrs-managed protocol buffer files.
+
+## Index
+
+* buffrs clean
```

Comment:
```suggestion
* [buffrs clean]()
```

Reply:
Is this really better? It gives off the false impression that the page exists, but the link actually points to the origin page.

Reply:
If it doesnt, you can resolve this, at least in the main summary it does.

I would remove this line if this is not a link tho! Its potentially confusing and could be read as a bug in the docs